eslint-plugin/README.md

```diff
@@ -59,6 +59,7 @@ Add `@ecocode` to the `plugins` section of your `.eslintrc`, followed by rules c
 | Name                                                                                   | Description                                                | ⚠️ |
 | :------------------------------------------------------------------------------------- | :--------------------------------------------------------- | :- |
 | [avoid-high-accuracy-geolocation](docs/rules/avoid-high-accuracy-geolocation.md)       | Avoid using high accuracy geolocation in web applications. | ✅  |
+| [no-css-animations](docs/rules/no-css-animations.md)                                   | Avoid usage of CSS animations                              | ✅  |
```

Comment:
```suggestion
| [avoid-css-animations](docs/rules/no-css-animations.md)                                   | Avoid usage of CSS animations                              | ✅  |
```
I think the rule prefix should be "avoid-" instead of "no-"